telemetry_core/src/main/java/com/newrelic/telemetry/metrics/CommonAttributesBuilder.java

```diff
@@ -0,0 +1,36 @@
+package com.newrelic.telemetry.metrics;
+
+import com.newrelic.telemetry.Attributes;
+
+/** Encapsulates the building of commonly used attributes. */
+public class CommonAttributesBuilder {
+
+  private Attributes attributes = new Attributes();
+  private String serviceName;
+  private String instrumentationProvider;
+
+  public CommonAttributesBuilder attributes(Attributes attributes) {
```

Comment:
detailed javadoc is needed throughout this class. This is a public API and should be documented as such. *Especially* the instrumentation provider, the meaning of which will not be obvious to end users.

Reply:
Sounds good, will do.